connect.go

```diff
@@ -19,16 +19,18 @@ func dial(network string, hosts []string, noDelay bool, r, w time.Duration, logf
 			return v
 		}
 		conn  net.Conn
-		index = abs(int(atomic.AddInt32(&tick, 1)))
+		ident = abs(int(atomic.AddInt32(&tick, 1)))
```

Comment:
Do you mean to use alt_host only for failover and try to connect to the first server of the list for each attempt to open driver?